README.md

```diff
@@ -30,25 +30,25 @@ issue #17][vcam#17].
 
 ## Build
 
-- Install QT
+- Install QT + dev packages
 
+Ubuntu/Debian:
 ```
-sudo apt install qtbase5-dev
+sudo apt install qtbase5-dev libobs-dev
 ```
 
-- Get obs-studio source code
-
+Fedora 32:
 ```
-git clone --recursive https://github.com/obsproject/obs-studio.git
+sudo dnf install obs-studio-libs qt5-qtbase-devel
 ```
 
-- Build plugins
+- Build plugin:
 
 ```
 git clone https://github.com/CatxFish/obs-v4l2sink.git
 cd obs-v4l2sink
 mkdir build && cd build
-cmake -DLIBOBS_INCLUDE_DIR="../../obs-studio/libobs" -DCMAKE_INSTALL_PREFIX=/usr ..
+cmake ..
```

Comment:
I had to specify the make install prefix as well like:

`-DCMAKE_INSTALL_PREFIX=/usr `

Otherwise the make install puts the plugin in  /usr/local/lib/obs-plugins/v4l2sink.so  which obs does not see on Fedora.

obs on Fedora appears to expect:

/usr/lib64/obs-plugins/v4l2sink.so

Reply:
https://github.com/CatxFish/obs-v4l2sink/pull/43/commits/798adeaf3dcf94d32984acb91fc2dc3fa9a708a5

added it back in ^^